src/java/main/org/apache/zookeeper/cli/ReconfigCommand.java

```diff
@@ -52,12 +52,12 @@
     private String members;
 
     /* version - version of config from which we want to reconfigure - if current config is different
-     * reconfiguration will fail. Should be ommitted from the CLI to disable this option.
+     * reconfiguration will fail. Should be committed from the CLI to disable this option.
```

Comment:
@maoling Thank you for the comment. Yep, `ommitted` should be changed into `committed`.